### PR TITLE
Refactor the About screen to show the QGIS and GDAL/OGR versions

### PR DESCRIPTION
**Pull request by suricactus**
Also small refactor on the `Label.text` implementation to bring it to the latest best practices in JavaScript:
- `var` -> `const`
- braces around `if`s

| before | after |
|-|-|
| image | image |

### CODE REVIEW

File: src/qml/About.qml [hunk 1]
```diff
@@ -74,10 +74,13 @@ Item {
...
             textFormat: Text.RichText
             text: {
-              var links = '<a href="https://github.com/opengisch/QField/commit/' + gitRev + '">' + gitRev.substr(0, 6) + '</a>';
-              if (appVersion && appVersion !== '1.0.0')
+              const links = '<a href="https://github.com/opengisch/QField/commit/' + gitRev + '">' + gitRev.substr(0, 6) + '</a>';
+              if (appVersion && appVersion !== '1.0.0') {
                 links += ' <a href="https://github.com/opengisch/QField/releases/tag/' + appVersion + '">' + appVersion + '</a>';
-              return "QField<br>" + appVersionStr + " (" + links + ")<br>Qt " + qVersion;
+              }
+              const dependencies = [["Qt", qVersion], ["QGIS", qgisVersion], ["GDAL/OGR", gdalVersion]];
```
Comment (suricactus) [inline src/qml/About.qml:81]: Open to change the order. Probably QGIS | GDAL | Qt makes more sense?

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 564039883baf9852fe49e6993c50a702e8cb3a34: [**arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-arm-neon.apk)
- [x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-x64.apk)
- [x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-x86.apk)
- [**all access arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-arm64.apk)
- [all access arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-arm-neon.apk)
- [all access x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-x64.apk)
- [all access x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5878-422c9f28437a75b25ec957fa873fe55c0459a9c2-android-x86.apk)
</details>

File: src/qml/About.qml (hunk 1 above)
Comment (suricactus) [reply src/qml/About.qml:81]: Done!

**Comment by suricactus** [comment]
@nirvn updated the code/screenshot with two changes:
- reordered the softwares and respective versions
- removed the name of the QGIS version

**Review by nirvn — APPROVED** [review]

**Comment by suricactus** [comment]
@nirvn there are quite some build failures. Is it because of my code or it is expected?

**Review by nirvn — APPROVED** [review]

**Pull request merged**